lucille-plugins/lucille-tika/src/main/java/com/kmwllc/lucille/tika/stage/TextExtractor.java

```diff
@@ -106,16 +125,44 @@ public Iterator<Document> processDocument(Document doc) throws StageException {
 
       String filePath = doc.getString(filePathField);
 
-      try {
-        InputStream inputStream = FileUtils.getInputStream(filePath);
-        parseInputStream(doc, inputStream);
-      } catch (IOException e) {
-        throw new StageException("InputStream cannot be parsed or created", e);
+      InputStream inputStream = getFileInputStream(filePath);
+      if (inputStream == null) {
+        log.warn("Error getting inputStream for file at : {}", filePath);
+        return null;
       }
+      parseInputStream(doc, inputStream);
     }
     return null;
   }
 
+  private InputStream getFileInputStream(String filePath) {
```

Comment:
The original FileUtils.getInputStream had extra functionality that we're not yet capturing here. If the filePath wasn't a valid URI it would treat it as a regular file path and open a FileInputStream...